kaldb/src/test/java/com/slack/kaldb/logstore/SchemaAwareLogDocumentBuilderImplTest.java

```diff
@@ -0,0 +1,737 @@
+package com.slack.kaldb.logstore;
+
+import static com.slack.kaldb.logstore.SchemaAwareLogDocumentBuilderImpl.*;
```

Comment:
Minor: after merging, this wildcard import will trigger the new errorprone rule that disallows wildcard imports.